ui/app/router.js

```diff
@@ -18,7 +18,8 @@ Router.map(function () {
       this.mount('open-api-explorer', { path: '/api-explorer' });
       this.route('license');
       this.route('clients', function () {
-        this.route('index', { path: '/' });
```

Comment:
Ember knows that `clients` is the `index` route so this is redundant (thanks @zofskeez for the ember lesson 🤓 )